### PR TITLE
Comments Pipelines

### PR DESCRIPTION
**Pull request by egolol**

### CODE REVIEW

File: braph2genesis/pipelines/connectivity directed/_AnalyzeEnsemble_CON_WD.gen.m [hunk 1]
```diff
@@ -60,37 +60,37 @@
 %% ¡props_update!
 
 %%% ¡prop!
-ELCLASS (constant, string) is the class of the % % % .
+ELCLASS (constant, string) is the class of the ensenmble-based graph analysis using connectivity directed data.
```
Comment (bzufiriaki) [inline braph2genesis/pipelines/connectivity directed/_AnalyzeEnsemble_CON_WD.gen.m]: "ensemble" --> check in all files

File: braph2genesis/pipelines/connectivity group_average/_AnalyzeGroup_CON_GA_WU.gen.m [hunk 2]
```diff
@@ -49,14 +49,14 @@
 %% ¡props_update!
 
 %%% ¡prop!
-ELCLASS (constant, string) is the class of the % % % .
+ELCLASS (constant, string) is the class of the graph analysis with averaged connectivity data.
```
Comment (bzufiriaki) [inline braph2genesis/pipelines/connectivity group_average/_AnalyzeGroup_CON_GA_WU.gen.m]: connectivity undirected data?

File: braph2genesis/pipelines/connectivity directed/_AnalyzeEnsemble_CON_WD.gen.m [hunk 3]
```diff
@@ -60,37 +60,37 @@
...
 %%%% ¡default!
 'AnalyzeEnsemble_CON_WD'
 
 %%% ¡prop!
-NAME (constant, string) is the name of the ensenmble-based graph analysis using connectivity data.
+NAME (constant, string) is the name of the ensenmble-based graph analysis using connectivity directed data.
```
Comment (bzufiriaki) [inline braph2genesis/pipelines/connectivity directed/_AnalyzeEnsemble_CON_WD.gen.m]: ensemble

File: braph2genesis/pipelines/connectivity directed/_AnalyzeEnsemble_CON_WD.gen.m [hunk 4]
```diff
@@ -60,37 +60,37 @@
...
 %%%% ¡default!
-'AnalyzeEnsemble_CON_WD'
+'Connectivity Weighted Directed Analyze Ensemble'
 
 %%% ¡prop!
-DESCRIPTION (constant, string) is the description of the ensenmble-based graph analysis using connectivity data.
+DESCRIPTION (constant, string) is the description of the ensenmble-based graph analysis using connectivity directed data.
```
Comment (bzufiriaki) [inline braph2genesis/pipelines/connectivity directed/_AnalyzeEnsemble_CON_WD.gen.m]: ensemble

File: braph2genesis/pipelines/connectivity directed/_AnalyzeEnsemble_CON_WD.gen.m [hunk 5]
```diff
@@ -60,37 +60,37 @@
...
 %%%% ¡default!
 'This ensemble-based graph analysis (AnalyzeEnsemble_CON_WD) analyzes connectivity data using weighted directed graphs.'
 
 %%% ¡prop!
-TEMPLATE (parameter, item) is the template of the ensenmble-based graph analysis using connectivity data.
+TEMPLATE (parameter, item) is the template of the ensenmble-based graph analysis using connectivity directed data.
```
Comment (bzufiriaki) [inline braph2genesis/pipelines/connectivity directed/_AnalyzeEnsemble_CON_WD.gen.m]: ensemble

File: braph2genesis/pipelines/connectivity directed/_AnalyzeEnsemble_CON_WD.gen.m [hunk 6]
```diff
@@ -60,37 +60,37 @@
...
 %%%% ¡settings!
 'AnalyzeEnsemble_CON_WD'
 
 %%% ¡prop!
-ID (data, string) is a few-letter code for the ensenmble-based graph analysis using connectivity data.
+ID (data, string) is a few-letter code for the ensenmble-based graph analysis using connectivity directed data.
```
Comment (bzufiriaki) [inline braph2genesis/pipelines/connectivity directed/_AnalyzeEnsemble_CON_WD.gen.m]: ensemble

File: braph2genesis/pipelines/connectivity directed/_AnalyzeEnsemble_CON_WD.gen.m [hunk 7]
```diff
@@ -60,37 +60,37 @@
...
 %%%% ¡default!
 'AnalyzeEnsemble_CON_WD ID'
 
 %%% ¡prop!
-LABEL (metadata, string) is an extended label of the ensenmble-based graph analysis using connectivity data.
+LABEL (metadata, string) is an extended label of the ensenmble-based graph analysis using connectivity directed data.
```
Comment (bzufiriaki) [inline braph2genesis/pipelines/connectivity directed/_AnalyzeEnsemble_CON_WD.gen.m]: ensemble

File: braph2genesis/pipelines/connectivity directed/_AnalyzeEnsemble_CON_WD.gen.m [hunk 8]
```diff
@@ -60,37 +60,37 @@
...
 %%%% ¡default!
 'AnalyzeEnsemble_CON_WD label'
 
 %%% ¡prop!
-NOTES (metadata, string) are some specific notes about the ensenmble-based graph analysis using connectivity data.
+NOTES (metadata, string) are some specific notes about the ensenmble-based graph analysis using connectivity directed data.
```
Comment (bzufiriaki) [inline braph2genesis/pipelines/connectivity directed/_AnalyzeEnsemble_CON_WD.gen.m]: ensemble

File: braph2genesis/pipelines/connectivity/_ImporterGroupSubjectCON_XLS.gen.m [hunk 9]
```diff
@@ -19,14 +19,14 @@ Subject ID (column 1), covariates (subsequent columns).
 %% ¡props_update!
 
 %%% ¡prop!
-ELCLASS (constant, string) is the class of the % % % .
+ELCLASS (constant, string) is the class of the CON subject group importer from XLS/XLSX.
```
Comment (bzufiriaki) [inline braph2genesis/pipelines/connectivity/_ImporterGroupSubjectCON_XLS.gen.m:22]: in some descriptions there is "... from XLS/XLSX" and in others there is "... from XLSX" or "... from XLS"

File: braph2genesis/pipelines/structural/_AnalyzeGroup_ST_BUD.gen.m [hunk 10]
```diff
@@ -66,14 +66,14 @@
 %% ¡props_update!
 
 %%% ¡prop!
-ELCLASS (constant, string) is the class of the % % % .
+ELCLASS (constant, string) is the class of the group-based graph analysis with structural data at fixed density.
```
Comment (bzufiriaki) [inline braph2genesis/pipelines/structural/_AnalyzeGroup_ST_BUD.gen.m:69]: sometimes is "**at** fixed density" and other times is "**of** fixed density"

**Review by bzufiriaki — CHANGES_REQUESTED** [review]

File: braph2genesis/pipelines/structural multiplex/_ImporterGroupSubjectST_MP_XLS.gen.m [hunk 11]
```diff
@@ -24,14 +24,14 @@ Subject ID (column 1), covariates (subsequent columns).
 %% ¡props_update!
 
 %%% ¡prop!
-ELCLASS (constant, string) is the class of the % % % .
+ELCLASS (constant, string) is the class of the ST MP subject group importer from XLS/XLSX.
```
Comment (hanzh346) [inline braph2genesis/pipelines/structural multiplex/_ImporterGroupSubjectST_MP_XLS.gen.m:27]: Should we use the full name "structural multiplex" on "ST MP", also it applies to other descriptions relevant.

**Pull request merged**